controllers/signupController.js

```diff
@@ -20,7 +22,15 @@ exports.signup = async (req, res) => {
     // Sprawdzenie, czy użytkownik o podanym emailu już istnieje
     let existingUser = await User.findOne({ email: req.body.email });
     if (existingUser) {
-      return res.status(409).json({ message: 'Email in use' });
+      return res.status(409).json({ message: 'Email already in use' });
+    }
+
+    // Generowanie avatarURL za pomocą Gravatara
+    const avatarURL = gravatar.url(req.body.email, { s: '200', d: 'retro' }, true);
+
+    // Możesz również dodać sprawdzenie czy avatarURL jest prawidłowy
+    if (!avatarURL.startsWith('http')) {
```

Comment:
Tutaj sprawdzasz czy gravatar.url poprawnie wykonał działanie, ale czy to nie jest zbyteczne? Czy teraz gravatar.url() może stworzyć link bez http?